gitmanager.py

```diff
@@ -315,7 +316,15 @@ def merge_pull_request(cls, pr_id, comment=""):
             raise ValueError("PR description is malformed. Blame a developer.")
         if pr_info["state"] != "open":
             raise ValueError("PR #{} is not currently open, so I won't merge it.".format(pr_id))
+        string = pr_info["title"]
         ref = pr_info['head']['ref']
+        string = regex.match(r".*?: \S+ (.*?)(?:\Z|\s*\(\?#)", str).group(1)
+        string = string.replace("\\", "")
+        is_watchlist = "watch" in pr_info["title"]
+        is_phone = string.isdigit()
```

Comment:
This is not a valid test for it being a number watch. There are plenty of number watches which include characters other than numbers. There are also times when we will watch numbers as keywords.